docs/splatalogue/splatalogue.rst

```diff
@@ -89,7 +89,7 @@ online interface.  In principle, we can make a higher level wrapper, but it is
 not obvious what other parameters one might want to query on (whereas with
 catalogs, you almost always need a sky-position based query tool).
 
-Any feature you can change on the `Splatalogue web form <splat_b>`_ can be
+Any feature you can change on the Splatalogue `query interface`_ can be
```

Comment:
Is there a reason for this change?  I think 'web form' is more descriptive than 'query interface' in this case.

Reply:
This link was broken so I used the named reference to http://www.cv.nrao.edu/php/splat/b.php . I will change the reference text to Splatalogue web form.